documentation/STRUCTURE.md

```diff
@@ -0,0 +1,161 @@
+
+## Required steps
+
+To properly format math, text, and links across the pages we need to use ReST. Some of our existing documentation may have to be converted from Markdown to ReST, e.g. the terminology page.
+
+We need to find a way to present the dashboard in a website where it can't be interactive. Perhaps with screenshots for the user guides, but the case studies are downloadable as Jupyter notebooks. [Could we perhaps pre-calculate all metrics and show the interactive dashboard in the case studies?]
+
+We already have the examples gallery thanks to Adrin's work on sphinx-gallery: https://fairlearn.readthedocs.io/en/latest/index.html
+There will be plenty of work to
+
+- convert existing notebooks to case studies (or user guides)
+- write all user guides
+- write the installation guide
+
+The repository structure should look similar to [what scikit-learn has](https://github.com/scikit-learn/scikit-learn/tree/master/doc):
+
+- top-level doc directory has all the ReST files for the webpage except for API documentation and case studies
+- API documentation comes directly from the code documentation
+- case studies live in a separate top-level directory (scikit-learn calls it `examples`) as python files
+
+Putting it all together:
+
+- deploy through fairlearn.org
+  - landing page will be provided by a designer, everything else from the Fairlearn project repositories
+  - CI should generate documentation and make it available at PR-time for reviewers to check
+  - all pages are reachable through fairlearn.org
+  - https for fairlearn.org (currently only http works)
+  - remove API doc from readthedocs
+- ensure the navigation from homepage to the other sections works
+  - manual testing?
+  - automated testing? (also check for broken links)
+
+## Development process
+
+We need to document all the processes around generating documentation. Specifically, we need to document how one can
+
+- build/generate the documentation and subsequently view it
+- develop case studies that end up in python files, but perhaps while creating them using Jupyter
+- get documentation changes into the Fairlearn repository (PR with Ci generating documentation and storing it as artifacts)
+
+## Outstanding questions
+
+1. Do we want users to cite us in any way? See [this example](https://scikit-learn.org/dev/about.html#citing-scikit-learn).
+1. Do we want user testimonials? It definitely provides credibility (assuming users are willing)
+1. Do we want a "News" section? It could list recent updates such as new versions (link to changelog), but also upcoming presentations, references to conference papers, blog posts, etc.
+1. Do we want a blog? [Example: pandas](https://pandas.pydata.org/community/blog/)
+1. Do we want to highlight differences to other fairness toolkits anywhere?
+1. Do we want some sort of "Getting started" page?
+1. Should we have a glossary? [Example: scikit-learn](https://scikit-learn.org/dev/glossary.html)
```

Comment:
we found that very useful.

Reply:
That's interesting feedback! I'll keep it down here because it's not the top priority as long as the rest of the documentation is lacking, but I'm hoping we'll get to it at some point :-)